service/controller.go

```diff
@@ -349,18 +353,19 @@ func (s *service) CreateVolume(
 
 	// Reading quota limit parameters
 	softLimit, advisoryLimit, softGracePrd = readQuotaLimitParams(params)
-	log.Infof("Limit parameters considered for quota creation SoftLimit: '%s' , AdvisoryLimit: '%s',SoftGracePrd: '%s'", softLimit, advisoryLimit, softGracePrd)
+	log.Infof("limit parameters considered for quota creation SoftLimit: '%s' , AdvisoryLimit: '%s',SoftGracePrd: '%s'", softLimit, advisoryLimit, softGracePrd)
```

Comment:
is this enforced by the checkers?

Reply:
Which checkers?

Reply:
start the sentence with a smallcase, for the log messages.

Reply:
No the checkers do not enforce this. I've seen the trend is for us to start log messages with lowercase so I've been changing in code I touch.